wwr_ui/randomizer_window.py

```diff
@@ -112,6 +112,7 @@ def __init__(self, cmd_line_args):
     self.set_option_description(None)
     
     self.update_settings()
+    self.update_health_label()
```

Comment:
I noticed that on startup the UI always showed "Current Starting Health: 3 Hearts" regardless of the value that is actually in the `settings.txt` file, this is a small fix for that